packages/frontend/core/src/components/root-app-sidebar/index.tsx

```diff
@@ -197,6 +197,16 @@ export const RootAppSidebar = ({
       });
   }, [blockSuiteWorkspace.id, navigateHelper, open, setting]);
 
+  const allPageActivePath = useMemo(() => {
+    if (
+      currentPath.startsWith(`/workspace/${currentWorkspaceId}/collection/`) ||
+      currentPath.startsWith(`/workspace/${currentWorkspaceId}/tag/`)
+    ) {
+      return paths.all(currentWorkspaceId);
+    }
+    return currentPath;
+  }, [currentPath, currentWorkspaceId, paths]);
```

Comment:
Please calculate the active status of the route links, and pass it to a new `active` prop to `RouteMenuLinkItem`